scripts/management/create_mgmt_db.py

```diff
@@ -38,14 +38,14 @@ def create_mgmt_db(realisations, f, srf_files=[]):
 
     db = initilize_db(f)    
     procs_to_be_done = get_procs(db)
-
```

Comment:
One more... 

In future it would be good to manually check your diff, to make sure you only have changes you want to be pushed.

Reply:
DONE. Good idea WILL DO